scripts/cipublish

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
The existing story around publishing thin OpenJDK 8 container images wrapping an OpenJDK 8 built JAR is defined here (the ECR part can be altered to support Quay, Docker Hub, etc.):

- https://github.com/azavea/raster-foundry/blob/develop/scripts/cipublish#L36-L73
- https://github.com/azavea/raster-foundry/blob/develop/app-backend/app/Dockerfile
- https://github.com/azavea/raster-foundry/blob/develop/app-backend/tile/.dockerignore